### ISSUE TITLE
Application crash while capture image

### ISSUE THREAD

**Issue by nahlanabil84** [akshay2211/PixImagePicker#5]
```
FATAL EXCEPTION: main
    Process:  PID: 28541
    java.lang.RuntimeException: autoFocus failed
        at android.hardware.Camera.native_autoFocus(Native Method)
        at android.hardware.Camera.autoFocus(Camera.java:1584)
        at com.wonderkiln.camerakit.Camera1$4.onTouch(Camera1.java:903)
        at android.view.View.dispatchTouchEvent(View.java:10009)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2665)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2344)
        at com.wonderkiln.camerakit.CameraView$2.onTouch(CameraView.java:170)
        at android.view.View.dispatchTouchEvent(View.java:10009)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2665)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2344)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2671)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2358)
        at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:447)
        at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1871)
        at android.app.Activity.dispatchTouchEvent(Activity.java:3213)
        at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:68)
        at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:409)
        at android.view.View.dispatchPointerEvent(View.java:10233)
        at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4865)
        at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4726)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4258)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4311)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4277)
        at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4404)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4285)
        at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4461)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4258)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4311)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4277)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4285)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4258)
        at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6690)
        at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6664)
        at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6625)
        at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6819)
        at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:356)
        at android.os.Looper.loop(Looper.java:138)
        at android.app.ActivityThread.main(ActivityThread.java:6577)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:832)
```

**Comment by akshay2211** [comment]
Hello @nahlanabil84,
Thank you for using **PIX**
this bug has been resolved in the latest version 1.0.5

**Comment by naseemali925** [comment]
Getting This On Image Capture @akshay2211 I Am Using Version1.0.5
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: ali.naseem.retrota, PID: 3839
                  java.lang.RuntimeException: setParameters failed
                      at android.hardware.Camera.native_setParameters(Native Method)
                      at android.hardware.Camera.setParameters(Camera.java:1964)
                      at com.wonderkiln.camerakit.Camera1$8.run(Camera1.java:1057)
                      at android.os.Handler.handleCallback(Handler.java:739)
                      at android.os.Handler.dispatchMessage(Handler.java:95)
                      at android.os.Looper.loop(Looper.java:145)
                      at android.app.ActivityThread.main(ActivityThread.java:6917)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at java.lang.reflect.Method.invoke(Method.java:372)
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
